src/test/regress/expected/multi_subquery.out

```diff
@@ -911,7 +911,7 @@ RESET citus.coordinator_aggregation_strategy;
 SELECT t1.event_type FROM events_table t1
 GROUP BY t1.event_type HAVING t1.event_type > corr(t1.value_3, t1.value_2 + (SELECT t2.value_2 FROM users_table t2 ORDER BY 1 DESC LIMIT 1))
 ORDER BY 1;
-ERROR:  result "68_1" does not exist
+WARNING: Query could not find the intermediate result file "X_XX", it was mostly likely deleted due to an error in a parallel process within the same distributed transaction
```

Comment:
related to https://github.com/citusdata/citus/pull/3537#issuecomment-601610294

Reply:
Let  me open an issue

Reply:
What do you think about this: #3709

That is not ideal, but we've had way too many issues around it, so I'm OK to get rid of the optimization